<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmCadastroCliente</class>
 <widget class="QMainWindow" name="FrmCadastroCliente">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>49</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>282</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>490</width>
    <height>282</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>490</width>
    <height>282</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Cadastra Cliente</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>imagens/icon-cachorro-marrom-30.png</normaloff>imagens/icon-cachorro-marrom-30.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(21, 143, 157);</string>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>611</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="4" column="0">
      <widget class="QLabel" name="cad_cli_txt_end02_label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 145, 10);
border-radius:0px</string>
       </property>
       <property name="text">
        <string>Endereço</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLineEdit" name="cad_cli_txt_fone_preferencial02">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(110, 165, 165);
border-radius:0px</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="maxLength">
        <number>14</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="cad_cli_txt_fone_alternativo_label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 145, 10);
border-radius:0px</string>
       </property>
       <property name="text">
        <string>Tel. Alternativo</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="13" column="0" colspan="6">
      <widget class="QPlainTextEdit" name="cad_cli_txt_observacao">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(110, 165, 165);
border-radius:0px</string>
       </property>
      </widget>
     </item>
     <item row="10" column="6">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="cad_cli_txt_nome02_label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 145, 10);
border-radius:0px</string>
       </property>
       <property name="text">
        <string>Nome</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="cad_cli_txt_complemento_label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 145, 10);
border-radius:0px</string>
       </property>
       <property name="text">
        <string>Complemento</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="3" colspan="3">
      <widget class="QLineEdit" name="cad_cli_txt_cidade02">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(110, 165, 165);
border-radius:0px</string>
       </property>
       <property name="maxLength">
        <number>40</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="cad_cli_txt_doc01_label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 145, 10);
border-radius:0px</string>
       </property>
       <property name="text">
        <string>CPF</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QLabel" name="cad_cli_txt_cidade02_label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 145, 10);
border-radius:0px</string>
       </property>
       <property name="text">
        <string>Cidade</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="cad_cli_txt_bairro02_label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 145, 10);
border-radius:0px</string>
       </property>
       <property name="text">
        <string>Bairro</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="cad_cli_txt_data_nasc0D">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(110, 165, 165);
border-radius:0px</string>
       </property>
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>10</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QLabel" name="cad_cli_txt_end_numero02_label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 145, 10);
border-radius:0px</string>
       </property>
       <property name="text">
        <string>Número</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLabel" name="cad_cli_txt_cepC_label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 145, 10);
border-radius:0px</string>
       </property>
       <property name="text">
        <string>CEP</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="cad_cli_txt_data_nasc0D_label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 145, 10);
border-radius:0px</string>
       </property>
       <property name="text">
        <string>Data. Nascimento</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="5">
      <widget class="QComboBox" name="comboBox_UF">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(110, 165, 165);
border-radius:0px</string>
       </property>
       <item>
        <property name="text">
         <string>AC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AM</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BA</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DF</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ES</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GO</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MA</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MG</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PA</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PR</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PI</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RJ</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RN</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RO</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RR</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TO</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="cad_cli_txt_doc01">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(110, 165, 165);
border-radius:0px</string>
       </property>
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="maxLength">
        <number>14</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::LogicalMoveStyle</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="5">
      <widget class="QLabel" name="comboBox_UF_label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 145, 10);
border-radius:0px</string>
       </property>
       <property name="text">
        <string>Estado</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="5">
      <widget class="QLineEdit" name="cad_cli_txt_nome02">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(110, 165, 165);
border-radius:0px</string>
       </property>
       <property name="maxLength">
        <number>62</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="5">
      <widget class="QLineEdit" name="cad_cli_txt_end02">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(110, 165, 165);
border-radius:0px</string>
       </property>
       <property name="maxLength">
        <number>62</number>
       </property>
      </widget>
     </item>
     <item row="14" column="2" colspan="3">
      <widget class="QPushButton" name="btn_cli_sair">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color: rgb(255, 255, 255);
background-color: rgb(255, 145, 10);
border-radius:0px
}

QPushButton:hover{
color:rgb(255, 145, 10);
background-color: rgb(255, 255, 255);
border-radius:0px
}
</string>
       </property>
       <property name="text">
        <string>Sair</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../imagens/icons-sinal-saída.png</normaloff>../../imagens/icons-sinal-saída.png</iconset>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QPushButton" name="btn_cli_cadastrar">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color: rgb(255, 255, 255);
background-color: rgb(255, 145, 10);
border-radius:0px
}

QPushButton:hover{
color:rgb(255, 145, 10);
background-color: rgb(255, 255, 255);
border-radius:0px
}
</string>
       </property>
       <property name="text">
        <string>Salvar</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../imagens/icon-add.png</normaloff>../../imagens/icon-add.png</iconset>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QLineEdit" name="cad_cli_txt_bairro02">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(110, 165, 165);
border-radius:0px</string>
       </property>
       <property name="maxLength">
        <number>40</number>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="cad_cli_txt_fone_preferencial02_label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 145, 10);
border-radius:0px</string>
       </property>
       <property name="text">
        <string>Tel. Preferêncial</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="cad_cli_txt_observacao_label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 145, 10);
border-radius:0px</string>
       </property>
       <property name="text">
        <string>Observações</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="11" column="3" colspan="3">
      <widget class="QLineEdit" name="cad_cli_txt_contato_alternativo">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(110, 165, 165);
border-radius:0px</string>
       </property>
       <property name="maxLength">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="cad_cli_txt_fone_alternativo">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(110, 165, 165);
border-radius:0px</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="maxLength">
        <number>14</number>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="3">
      <widget class="QLabel" name="cad_cli_txt_contato_alternativo_label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 145, 10);
border-radius:0px</string>
       </property>
       <property name="text">
        <string>Nome do Contato</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="5">
      <widget class="QLineEdit" name="cad_cli_txt_complemento">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(110, 165, 165);
border-radius:0px</string>
       </property>
       <property name="maxLength">
        <number>62</number>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QLineEdit" name="cad_cli_txt_end_numero02">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(110, 165, 165);
border-radius:0px</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="maxLength">
        <number>5</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QLineEdit" name="cad_cli_txt_cepC">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(110, 165, 165);
border-radius:0px</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="maxLength">
        <number>9</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>291</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="1" colspan="2">
      <widget class="QPushButton" name="btn_abre_formcapturaimagem">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color: rgb(255, 255, 255);
background-color: rgb(255, 145, 10);
border-radius:0px
}

QPushButton:hover{
color:rgb(255, 145, 10);
background-color: rgb(255, 255, 255);
border-radius:0px
}
</string>
       </property>
       <property name="text">
        <string>Foto</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../imagens/icons8-câmera-24.png</normaloff>../../imagens/icons8-câmera-24.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="imagemcamera_label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>500</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(110, 165, 165);
border-radius:0px</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cad_cli_txt_doc01</tabstop>
  <tabstop>cad_cli_txt_data_nasc0D</tabstop>
  <tabstop>cad_cli_txt_nome02</tabstop>
  <tabstop>cad_cli_txt_cepC</tabstop>
  <tabstop>cad_cli_txt_end02</tabstop>
  <tabstop>cad_cli_txt_end_numero02</tabstop>
  <tabstop>cad_cli_txt_complemento</tabstop>
  <tabstop>comboBox_UF</tabstop>
  <tabstop>cad_cli_txt_bairro02</tabstop>
  <tabstop>cad_cli_txt_cidade02</tabstop>
  <tabstop>cad_cli_txt_fone_preferencial02</tabstop>
  <tabstop>cad_cli_txt_fone_alternativo</tabstop>
  <tabstop>cad_cli_txt_contato_alternativo</tabstop>
  <tabstop>cad_cli_txt_observacao</tabstop>
  <tabstop>btn_cli_cadastrar</tabstop>
  <tabstop>btn_cli_sair</tabstop>
  <tabstop>btn_abre_formcapturaimagem</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
